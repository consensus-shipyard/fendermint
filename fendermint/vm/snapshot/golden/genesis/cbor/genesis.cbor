a4666865696768741bffffffffffffffff6473697a651bfcc72e180b56c57b666368756e6b73006c73746174655f706172616d73a76a73746174655f726f6f74d82a56000138e49ca0730ea0285e6c6918e9c64863d0b3f4046974696d657374616d701bc411f28f1c0667d56f6e6574776f726b5f76657273696f6e1affffffff68626173655f666565510063c1e78f40b7addef0026063feb02af66b636972635f737570706c7951001c87223d02257803429bfc02d2a0138e68636861696e5f69641b000d78353f3d56bc6b706f7765725f7363616c6500