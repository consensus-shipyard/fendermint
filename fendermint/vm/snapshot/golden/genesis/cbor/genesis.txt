SnapshotManifest { height: 18446744073709551615, size: 18214577898727589243, chunks: 0, state_params: FvmStateParams { state_root: Cid(bae4ojhfaomhkakc6nrurr2ogjbr5bm7uaq), timestamp: Timestamp(14128340202502383573), network_version: NetworkVersion(4294967295), base_fee: TokenAmount(132600381462678671360.880266302272318198), circ_supply: TokenAmount(37920038393388550717.163537017434280846), chain_id: 3791344786822844, power_scale: 0 } }